frontend/src/components/modules/GameInterface/PointTable.tsx

```diff
@@ -224,7 +224,7 @@ const PointTable: React.FC<TableComponentProps> = ({ matchInfo }) => {
                 width: "20px",
                 height: "20px",
                 borderRadius: "4px",
-                backgroundColor: "#db4744",
+                backgroundColor: "#D01C1C",
```

Comment:
where is this new color coming from? Tbh now that I checked it could work better with new one